api/api.go

```diff
@@ -62,6 +62,7 @@ const (
 	SpecExportOptions        = "export_options"
 	SpecExportOptionsEmpty   = "empty_export_options"
 	SpecMountOptions         = "mount_options"
+	SpecFaCreateOptions      = "createoptions"
```

Comment:
please follow the naming convention of other variables in the spec-
`fa_creation_options`

Reply:
ok,  please document that as a comment above this line. Mention that this spec key cannot change due to parity with existing PSO storageclasses

Reply:
Sure